src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -112,32 +116,23 @@ export const CustomerInstitutionMetadataForm = ({
                         },
                         doiAgent: emptyProtectedDoiAgent,
                       });
+                      setCustomerData(selectedInstitution);
                     }}
                     errorMessage={touched && !!error ? error : undefined}
                     fieldInputProps={field}
                   />
-                ) : (
-                  <TextField
-                    variant="filled"
-                    label={t('common.institution')}
-                    data-testid={dataTestId.organization.searchField}
-                    required
-                    disabled
-                    {...field}
-                  />
-                )
-              }
-            </Field>
-            <CustomerInstitutionTextField
-              name={CustomerInstitutionFieldNames.DisplayName}
-              label={t('basic_data.institutions.display_name')}
-              required
-              dataTestId={dataTestId.basicData.institutionAdmin.displayNameField}
-            />
-            <CustomerInstitutionTextField
-              name={CustomerInstitutionFieldNames.ArchiveName}
-              label={t('basic_data.institutions.archive_name')}
-              dataTestId={dataTestId.basicData.institutionAdmin.archiveNameField}
+                )}
+              </Field>
+            )}
+
+            <CustomerInstitutionInformationFromCristin
+              customerData={customerData}
+              cristinId={values.customer.cristinId}
+              setName={(newName) => {
+                setFieldValue(CustomerInstitutionFieldNames.DisplayName, newName);
+                setFieldValue(CustomerInstitutionFieldNames.Name, newName);
+              }}
```

Comment:
`CustomerInstitutionInformationFromCristin` virker for meg som en "dum" komponent som bare skal vise data fra cristin, og stusser over at den også skal ha ansvaret for å sette noe data 🤔 
Skal ikke dette heller håndteres av `onChange` til `<OrganizationSearchField>` litt over? Virker naturlig at dette bare skal endres når bruker velger en ny cristin-organisasjon, og at denne komponenten bare skal vise data, uten å vite så mye mer om hva som skjer rundt seg

Reply:
Problemet er hvis en kunde endrer navn i Cristin. Da har vi ikke mulighet til å endre navnet i lista over kunder i nva.